WeakAuras/WeakAuras.lua

```diff
@@ -193,6 +193,9 @@ function SlashCmdList.WEAKAURAS(input)
     Private.PrintHelp();
   elseif msg == "repair" then
     StaticPopup_Show("WEAKAURAS_CONFIRM_REPAIR", nil, nil, {reason = "user"})
+  elseif msg == "optin" then
```

Comment:
note that I didn't add this command to the help message, because I don't think we need to be advertising possibly unstable features to random users - if we wanted to use this we would ping ppl with the WAalpha role in discord instead IMO.

Reply:
do we still want/need the optin system? It could be nice if we want some user to test some specific feature and just go /optin FeatureXNAME